src/components/LandingPage/index.tsx

```diff
@@ -50,7 +50,7 @@ const LandingPage: React.FunctionComponent<{}> = () => {
                         related plots.
                     </p>
                     <div className={styles.cards}>
-                        {TRAJECTORIES.map((trajectory) => {
+                        {TRAJECTORIES.slice(0, 2).map((trajectory) => {
```

Comment:
probably overkill, but it might be a good idea to make a variable named "NUMBER_OF_CARDS_PER_ROW" and make this NUMBER_OF_CARDS_PER_ROW-1, just so if we ever want to change the layout we know what value to change.